bot.ts

```diff
@@ -275,7 +317,20 @@ export class Bot {
       }
     } catch (error) {
       logger.error({ mint: rawAccount.mint.toString(), error }, `Failed to sell token`);
+      if (trade) {
+        trade.close(0, 0, 'sell_failed');
+        this.balance += trade.profit;
```

Comment:
This is not really needed since we later on call `this.updatebalance`, but that's paving the way for a future dry run mode implementation.